pkg/tests/api/correlations/correlations_create_test.go

```diff
@@ -211,7 +211,7 @@ func TestIntegrationCreateCorrelation(t *testing.T) {
 		require.NoError(t, res.Body.Close())
 	})
 
-	t.Run("Should correctly create a correlation", func(t *testing.T) {
+	t.Run("Should correctly create a correlation without a config", func(t *testing.T) {
 		description := "a description"
```

Comment:
Did we want to test that config is null here?

Reply:
Yes, it's the current frontend behavior (config is not passed but we want to create a default config object so it's still possible to create correlations in the frontend).